projects/centre_console/test/test_charging_manager.c

```diff
@@ -0,0 +1,82 @@
+#include "can_transmit.h"
+#include "can_unpack.h"
+#include "centre_console_events.h"
+#include "charging_manager.h"
+#include "drive_fsm.h"
+#include "exported_enums.h"
+#include "ms_test_helpers.h"
+#include "test_helpers.h"
+
+#define TEST_CAN_DEVICE_ID 0x1
+
+static CanStorage s_can_storage;
+static CanSettings s_can_settings = {
+  .device_id = TEST_CAN_DEVICE_ID,
+  .bitrate = CAN_HW_BITRATE_500KBPS,
+  .rx_event = CENTRE_CONSOLE_EVENT_CAN_RX,
+  .tx_event = CENTRE_CONSOLE_EVENT_CAN_TX,
+  .fault_event = CENTRE_CONSOLE_EVENT_CAN_FAULT,
+  .tx = { GPIO_PORT_A, 12 },
+  .rx = { GPIO_PORT_A, 11 },
+  .loopback = true,
+};
+
+static DriveState s_drive_state;
+
+void setup_test(void) {
+  s_drive_state = NUM_DRIVE_STATES;
+  event_queue_init();
+  can_init(&s_can_storage, &s_can_settings);
+  TEST_ASSERT_OK(init_charging_manager(&s_drive_state));
+}
+
+void teardown_test(void) {}
+
+void test_get_global_charging_state(void) {
+  ChargingState *charging_state = get_global_charging_state();
+  TEST_ASSERT(*charging_state == CHARGING_STATE_NOT_CHARGING);
```

Comment:
Shouldn't these be `TEST_ASSERT_EQUAL` instead?

Reply:
true